question5.py

```diff
@@ -0,0 +1,11 @@
+numbers = [9,4,8,10,2,4,8,3,14,4,8]
+
+num = 12
+
+pairs = []
+
+for i in range(len(numbers)):
+    for j in range(len(numbers)):
+        if numbers[i]+numbers[j]== num and sorted([numbers[i],numbers[j]]) not in pairs:
```

Comment:
Make sure you have code formatted properly. Before and after `==`, there must be one space.

It is good to put `sorted([numbers[i],numbers[j]])` in a variable, as you are using it twice.